Sources/MockoloFramework/Utils/InheritanceResolver.swift

```diff
@@ -57,10 +60,11 @@ func lookupEntities(key: String,
             // If the protocol inherits other protocols, look up their entities as well.
             for parent in current.entityNode.inheritedTypes {
                 if parent != .class, parent != .anyType, parent != .anyObject {
-                    let (parentModels, parentProcessedModels, parentAttributes, parentPaths, parentPathToContents) = lookupEntities(key: parent, declType: declType, protocolMap: protocolMap, inheritanceMap: inheritanceMap)
+                    let (parentModels, parentProcessedModels, parentAttributes, parentInheritedTypes, parentPaths, parentPathToContents) = lookupEntities(key: parent, declType: declType, protocolMap: protocolMap, inheritanceMap: inheritanceMap)
                     models.append(contentsOf: parentModels)
                     processedModels.append(contentsOf: parentProcessedModels)
                     attributes.append(contentsOf: parentAttributes)
+                    inheritedTypes = inheritedTypes.union(parentInheritedTypes)
```

Comment:
please use `formUnion` to avoid CoW overhead

Reply:
Thank you for the review! 
I was not aware of the issue with Cow overhead. I will fix this along with the other review points.